### DISCUSSION TITLE
Scheduled Reminder respond 500 when `Your PR has failed CI checks` input has long text

### DISCUSSION THREAD

**Discussion by sisisin** [community/community#38754] [Actions]
### Select Topic Area

Bug

### Body

Scheduled Reminder respond 500 when `Your PR has failed CI checks` has long text
https://github.com/settings/reminders/

sample input

```
check-my-application1,check-my-application2,check-my-application3,check-my-application4,check-my-application5,check-my-application6,check-my-application7,check-my-application8,check-my-application9,check-my-application10,check-my-application11,check-my-application12,check-my-application13,check-my-application14,check-my-application15,check-my-application16,check-my-application17,check-my-application18,check-my-application19,check-my-application20,check-my-application21,check-my-application22,check-my-application23,check-my-application24,check-my-application25,check-my-application26,check-my-application27,check-my-application28,check-my-application29,check-my-application30,check-my-application31,check-my-application32,check-my-application33,check-my-application34,check-my-application35,check-my-application36,check-my-application37,check-my-application38,check-my-application39,check-my-application40,check-my-application41,check-my-application42,check-my-application43,check-my-application44,check-my-application45,check-my-application46,check-my-application47,check-my-application48,check-my-application49,check-my-application50
```

**Comment by ossanna16** [comment]
Hi there @sisisin and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
